polkadot/xcm/xcm-simulator/fuzzer/src/fuzz.rs

```diff
@@ -197,8 +228,20 @@ fn run_input(xcm_messages: [XcmMessage; 5]) {
 		}
 		#[cfg(not(fuzzing))]
 		println!();
+		// We run integrity tests and try_runtime invariants
+		[ParaA::execute_with, ParaB::execute_with, ParaC::execute_with].iter().for_each(
+			|execute_with| {
+				execute_with(|| {
+					parachain::AllPalletsWithSystem::try_state(Default::default(), All).unwrap();
```

Comment:
`All` also tries to decode the storage every time, it should be very slow once the state gets bigger.  
Maybe its enough to call this only every 100th time or so, depending on performance.

Reply:
To have optimal results, a fuzzer needs stability (each input should yield the same code coverage every time they are run). It would probably be counter-productive to do some kind of modulo 100 on try_state.

The harness is fast and the coverage is good, I don't think this is an issue :)